test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="librarytest/php-fpm-hello-web:fcgi"
```

Comment:
Oh and you should switch this to single quotes while you're there -- unnecessary interpolation and all that fun.